src/FieldNation/TypeValidator.php

```diff
@@ -73,7 +73,7 @@ public static function validateWorkOrderSerializerInterface(WorkOrderSerializerI
         self::validate('bool', $wo->getIsPrintable(), 'validatePrimitive', true, 'isPrintable');
 
         // Optional Fields
-        self::validate(GroupInterface::class, $wo->getGroup(), 'validateType', false, 'group');
```

Comment:
Oh I see your comment in the PR now. I see how it should have been used. Looks good